templates/tab/js/m365/package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fluentui/react-northstar": "^0.60.1",
+    "@fluentui/react-northstar": "^0.62.0",
```

Comment:
What's this change for?

Reply:
msteams-react-base-component depends on `msteams-react-base-component@^0.62.0`.